### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by Flova**
## Proposed changes
- Add TVM documentation
- Update ROS 1 vs ROS 2 stuff

### CODE REVIEW

File: bitbots_vision/docs/manual/tutorials/tvm.rst [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+    .. code-block:: bash
+
+        pip install onnx onnxsim xgboost
+
+
+How to optimize a model
+------------------------
+
+- Run the following commands if you are anoyed by deprication warnings
```
[the remark this answers is not in the document]
Comment (Flova) [reply bitbots_vision/docs/manual/tutorials/tvm.rst:42]: Oh, I will put it through spell check. I think it was off in my vs code :/

File: bitbots_vision/docs/manual/tutorials/tvm.rst [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+        .. code-block:: bash
+
+            tvmc run <model_name>.tar --profile --print-time --device "vulkan" --repeat 100
+
+    The command shows you a profiling of each layer. Check if they all run on ``vulkan0``. At the bottom a timing benchmark is printed.
+
+Run the model using the Python API
+----------------------------------
+
+Extract the tar, using `tar -xf <your_file>`.
```
Comment (jaagut) [inline bitbots_vision/docs/manual/tutorials/tvm.rst:105]:
```suggestion
Extract the tar, using ``tar -xf <your_file>``.
```

File: bitbots_vision/docs/manual/tutorials/tvm.rst [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+        # Run the network
+        module.set_input(InputLayer=input_data)
+        module.run()
+        yolo_detections, segmentation = module.get_output(0).numpy(), module.get_output(1).numpy()
+
+        print(yolo_detections.shape, segmentation.shape)
+
+
+Compile a YOEO model directly
```
Comment (jaagut) [inline bitbots_vision/docs/manual/tutorials/tvm.rst:135]: The existence of these tools should probably be mentioned in the beginning, or else one runs all the commands manually before noticing this.

File: bitbots_vision/docs/manual/tutorials/tvm.rst [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+
+        .. code-block:: bash
+    
+            tvmc compile --target "vulkan" --target-vulkan-from_device 0 --output <output_name>.tar --tuning-records <optimization_run_name>.json <onnx_file>
+
+    - You also want to do this on the "NUC" because the model will be optimized for the specific hardware.
+    - This should **not** take hours.
+    - Remember to replace all placeholders in the command
+    - You will hopefully end up with a ``.tar`` file containing the following items 
+    -
```
Comment (jaagut) [inline bitbots_vision/docs/manual/tutorials/tvm.rst:86]:
```suggestion
```

**Review by jaagut — APPROVED** [review]

**Pull request merged**